orchestra/static/orchestra/team-info/team-info-card.html

```diff
@@ -33,7 +33,18 @@ <h3>
           <tbody>
             <tr ng-repeat="assignment in teamInfoCard.assignments">
               <td>{{assignment.role}}</td>
-              <td>{{assignment.worker.username}}</td>
+              <td>{{assignment.worker.username}}
+                <button
+									class="btn btn-default btn-xs pull-right dsu-pl-30"
+                  ng-if="teamInfoCard.isProjectAdmin && assignment.status !== 'Complete'"
```

Comment:
Assignment statuses don't have a `Complete` state: https://github.com/b12io/orchestra/blob/main/orchestra/models/core/models.py#L440

Perhaps we want to check if the task (not assignment) status is not complete or aborted: https://github.com/b12io/orchestra/blob/main/orchestra/models/core/models.py#L369

Reply:
Renamed :)

Reply:
And add a helper :)